### ISSUE TITLE
Create Release [Start by 25/01]

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#6019]
As a developer I want a IBEX release so I can install it on the instrument machines for the next cycle, 16th Feb. This will be a major release as it contains dependency updates and Python 3 conversions so we may want to discuss which instruments we release to given the increased risk.

Acceptance Criteria:

- [ ] [Pre Testing Steps](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Creating-a-release#pre-testing)
- [ ] [Testing steps](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Creating-a-release#testing)
- [ ] [Post Testing](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Creating-a-release#post-testing)
- [ ] We decide where to deploy to and an appropriate deployment ticket is created
- [ ] [Create a ticket to update dependencies to latest versions and test](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Dependency-updates)
 - [ ] [Create a ticket to give instrument demos](https://github.com/ISISComputingGroup/IBEX/wiki/Timetable-for-Instrument-Demos)
 - [ ] Distribute release notes via email [Instrument scientist release email](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Updating-Instrument-Machines#a-new-release)

**Comment by kjwoodsISIS** [comment]
We should bear in mind that MuSR is a migration target - the Muons group likes to get new releases well in advance (at least 2 weeks) of the start of cycle.

**Comment by DominicOram** [comment]
Date likely to change given delay in cycle. Don't start this until we've discussed the fallout as a group.

**Comment by John-Holt-Tessella** [comment]
Not needed because cycle has been postponed

**Comment by rerpha** [comment]
Just as a note we have created [7.3.0](https://github.com/ISISComputingGroup/IBEX/issues/6148) but the plan is we'll create another release before the next cycle, presumably 7.4.0 unless there are breaking changes that get included between now and then

**Comment by rerpha** [comment]
Next release ticket: https://github.com/ISISComputingGroup/IBEX/issues/6149
